lib/controller.php

```diff
@@ -185,6 +185,14 @@ public function import_blob($blob) {
 			'_sha'         => $blob->sha,
 		) );
 
+		if(class_exists("WPCom_Markdown")) {
+			$wpcomMd = WPCom_Markdown::get_instance();
+			$args = $wpcomMd -> wp_insert_post_data($args, array(
+				"ID"           => $this->id,
+				"post_type"    => $args['post_type']
```

Comment:
I don't think we're exporting the `post_type` so `$args['post_type']` will throw a Undefined Offset notice.